rustic-rustfmt.el

```diff
@@ -188,6 +196,29 @@ and it's `cdr' is a list of arguments."
         (kill-buffer proc-buffer)
         (message "Workspace formatted with cargo-fmt.")))))
 
+;;;###autoload
+(defun rustic-format-region ()
+  "Format the current active region using rustfmt.
+
+This operation requires a nightly version of rustfmt.
+"
+  (interactive)
+  (unless (or (eq major-mode 'rustic-mode)
+              (eq major-mode 'rustic-macro-expansion-mode))
+    (error "Not a rustic-mode buffer."))
+  (let ((file (buffer-file-name (current-buffer)))
```

Comment:
Maybe it makes sense to use a variable for the current buffer since we use it twice.

Reply:
good point!